docs/guide/cloud_concepts/first-steps.md

```diff
@@ -0,0 +1,132 @@
+# Prefect Cloud: First Steps
+
+Welcome to Prefect Cloud! Now that you have an account, this guide will help you orient yourself and understand the minor adjustments required to deploy your Flows to Cloud.
+
+## Prefect Agent
+
+_Documentation Coming Soon!_
+
+## Create a Project
```

Comment:
`client.login` will need to be called prior to this. Either directly through code or through the CLI command `prefect auth login`

Reply:
^^ this is addressed in the previous section